kde-apps/kdepim-runtime/kdepim-runtime-9999.ebuild

```diff
@@ -64,15 +62,19 @@ RDEPEND="
 	>=kde-frameworks/kwallet-${KFMIN}:6
 	>=kde-frameworks/kwidgetsaddons-${KFMIN}:6
 	>=kde-frameworks/kwindowsystem-${KFMIN}:6
-	>=kde-frameworks/kxmlgui-${KFMIN}:6
```

Comment:
```
akonadi_davgroupware_resource => /usr/bin/akonadi_davgroupware_resource (interpreter => /lib64/ld-linux-x86-64.so.2)
    libQt6Xml.so.6 => /usr/lib64/libQt6Xml.so.6
```

Looks like it is still required:

=> `grep -Ri xmlgui: agents/newmailnotifier/CMakeLists.txt:    KF6::XmlGui`